### DISCUSSION TITLE
2024

### DISCUSSION THREAD

**Discussion by edonyzpc** [edony-ink/comments#70] [General]
🚀

**Comment by dong2019** [comment]
根据全文来看，我觉得博主您的年终总结不能够这么简单，或许在您的电脑本地有一个更加丰富的总结呢。

我从2017年开始写年终总结，现在越写越长了都。

今年的年终总结超乎想象，花了13h，写了89页，感觉就是人生重新过了一遍。

如果您有兴趣的话，我们可以交流交流关于年终总结的话题

**Reply by edonyzpc** [reply]
@dong2019 感谢你的回复与交流！

注意到你从17年就已经开始做年终总结，而且还能坚持用13个小时来梳理和反思，这一点非常令我敬佩也值得我学习。

说到我自己的年度总结，我还是会觉得给订阅者发自己个人的年度总结有点羞耻，因为我不确定我偏私人的东西对读者是不是一种打扰！所以，对于你的回复我感到非常荣欣也很感激。

说回年度总结，我的做法是周期性的，不会把所有的总结都堆到年底，一方面家庭和工作的关系确实时间上不允许，另一方面一整年的时间才反思和总结，我还是会有遗漏，所以我会阶段性的把自己的大脑记录下来，或发布、或留存，例如：
- [我的消费模型](https://www.edony.ink/my-consumer-decision-model/)
- [深入思考问题方法的探索](https://www.edony.ink/deep-thinking-on-complex-problems/)
- ...

至于年度总结我还是问题驱动式的引导自己去总结和回顾。

_P.S._ 我相信你的总结一定有很多好东西值得学习和借鉴，不知道可否有链接分享
_P.P.S._ 如果你有什么话题或者想法欢迎私信交流🥳

**Reply by dong2019** [reply]
从您网站的归档文件中看到了您在2024年的思考及相关的总结。非常感谢。

特别是关于消费模型，非常好，未来可能要借鉴下您的这篇文章

至于我的年终总结的话，涉及到非常多的细节内容，放到网络等于把自己完完全全暴露在网络中，所以这个暂时不能够给您看。一般呢，我是发给我认识的人看。

这个年终总结，是以《奇特的一生》为理论指导，以每日的日记为基础材料，整理汇总而成。当然，日记记录和总结的方法都已经写出来了都，您可以看看有没有啥值得借鉴之处。

1. 2022.12.16 [029 快六年的日记写作感受](http://ddw2019.com/029)
2. 2023.01.18 [040 扫描完纸质日记之后再读《奇特的一生》](http://ddw2019.com/040)
3. 2023.01.23 [041 过年期间阅读”2022日记总结“的感受](http://ddw2019.com/041)
4. 2024.01.23 [068 如何通过日记实现个人精进](http://ddw2019.com/068)
5. 2024.01.26 [S001 2022年第四季度的日记模板](http://ddw2019.com/s001)
6. 2024.12.25 [158 认知升级指南：如何通过在“人生操作系统”上加装“外挂软件”来升级你的认知](http://ddw2019.com/158)
7. 2023.01.03 [031 “道德完善计划”是什么？有何影响？怎么做？](http://ddw2019.com/031)

如果这篇文章对您有一点帮助，就是我的荣幸。

不过更希望你看的是《奇特的一生》，虽然这本书我16年第一遍看的时候相当之枯燥。不过越看越有意思。目前来说，已经看了不下一百遍。每一次都能够读出来不懂的东西

1. 时间开销日记
2. 任务规划的特征
3. 图片管理的标注
4. 如何保持简约
5. 如何管理文档以及他们之间的关系
6. ……

当然了这些内容都并非这本书的重点，只是一部分细节提醒了我。然后自己再去找相关的图书看看。

当然更多的非日记的内容，你可以访问我的[Dongdong‘ Blog](https://ddw2019.com)。

非常高兴认识您。刚刚在你的网站的首页底部的rss链接点进去了 链接为

https://feedly.com/i/subscription/feed%2Fhttps%3A%2F%2Fwww.edony.ink%2Frss%2F

但是复制到我的rss阅读器Inoreader中的时候，发现其出现的并不是您的博客文章，不知如何进行处理。
